app/src/main/java/framgia/com/ichat/AplicationAppGlideModule.java

```diff
@@ -0,0 +1,8 @@
+package framgia.com.ichat;
+
+import com.bumptech.glide.annotation.GlideModule;
+import com.bumptech.glide.module.AppGlideModule;
+
+@GlideModule
+public class AplicationAppGlideModule extends AppGlideModule {
```

Comment:
Application và App thì khác gì nhau 
```suggestion
public class ApplicationGlideModule extends AppGlideModule {
```